AdminApi.Sdk/Endpoints/ApiKeyValidation/ApiKeyValidationEndpoint.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backbone.AdminApi.Sdk.Endpoints.ApiKeyValidation.Types;
+using Backbone.AdminApi.Sdk.Endpoints.Common;
+using Backbone.AdminApi.Sdk.Endpoints.Common.Types;
+
+namespace Backbone.AdminApi.Sdk.Endpoints.ApiKeyValidation;
+
+public class ApiKeyValidationEndpoint(EndpointClient client) : Endpoint(client)
+{
+    public async Task<AdminApiResponse<ValidateApiKeyResponse>> ValidateApiKey(ValidateApiKeyRequest? request)
```

Comment:
Why is there the authenticated version of the request? I don't think that makes any sense, does it? ;)